docker/cmd/adaptive-metrics/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"log"
+	"os"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatalf("missing command, available commands: pull, apply")
+	}
+
+	switch os.Args[1] {
+	case "pull-recommendations":
```

Comment:
Should this be just `pull`?

Reply:
[aab6cc6](https://github.com/grafana/adaptive-metrics-autoapply/pull/12/commits/aab6cc625d45da60d4c0ca91e748ea52b32054c5)